env-files/account.template.env

```diff
@@ -30,12 +30,14 @@ ACCOUNT_WEBHOOK_PORT=3001
 WEBHOOK_BASE_URL=http://host.docker.internal:3001/webhooks/account-service
 
 # Redis URL
+# Only one of REDIS_URL or REDIS_OPTIONS may be specified.
 REDIS_URL=redis://0.0.0.0:6379
 #REDIS_URL=redis://redis:6379                        #docker dev
 
-# Additional Redis client options as a JSON object string
-# See https://github.com/redis/ioredis/blob/2ed6414339bdda3e427ed1e8484263cc4848f2ac/lib/redis/RedisOptions.ts
-REDIS_OPTIONS='{"commandTimeout":5000}'
+# Redis client options as a JSON object string.
+# Only one of REDIS_URL or REDIS_OPTIONS may be specified.
+# See https://ioredis.readthedocs.io/en/latest/API/#new-redisport-host-options
+#REDIS_OPTIONS='{"commandTimeout":10000}'
```

Comment:
We should give in the example setting the redis host since that would be required since you cannot use the REDIS_URL.

Reply:
Updated.